### PR TITLE
Added Quickplay Data

### PR DESCRIPTION
**Pull request by SLLCoding**

### CODE REVIEW

File: quickplay-games.json [hunk 1]
```diff
@@ -1,241 +1,231 @@
...
+				{ "name": "Solo NoDebuffs", "command": "/play duels_potion_duel", "enabled": true },
+				{ "name": "Solo Combo", "command": "/play duels_combo_duel", "enabled": true },
+				{ "name": "Solo Potion", "command": "/play duels_potion_duel", "enabled": false },
+				{ "name": "Solo OP", "command": "/play duels_op_duel", "enabled": true },
+				{ "name": "Doubles OP", "command": "/play duels_op_doubles", "enabled": true },
+				{ "name": "Solo Mega Walls", "command": "/play duels_mw_duel", "enabled": true },
+				{ "name": "Doubles Mega Walls", "command": "/play duels_mw_doubles", "enabled": true },
+				{ "name": "Solo Sumo", "command": "/play duels_solo_sumo", "enabled": true },
+				{ "name": "Solo Blitz", "command": "/play duels_blitz_duel", "enabled": true },
+				{ "name": "Solo Bow Spleef", "command": "/play duels_bowspleef_duel", "enabled": false },
```
Comment (SLLCoding) [inline quickplay-games.json:178]: Disabled or removed?

File: quickplay-games.json [hunk 2]
```diff
@@ -1,241 +1,231 @@
...
 			"icon": "https://hypixel.net/styles/hypixel-uix/hypixel/game-icons/Prototype-64.png",
 			"modes": [
-				{ "name": "Capture The Wool", "command": "/play prototype_pvp_ctw" },
-				{ "name": "King of the Hill 6v6", "command": "/play prototype_koth_two_team" },
-				{ "name": "King of the Hill 3v3v3v3", "command": "/play prototype_koth_four_team" },
-				{ "name": "TowerWars Solo", "command": "/play prototype_towerwars_solo" },
-				{ "name": "TowerWars Teams", "command": "/play prototype_towerwars_team_of_two" }
+				{ "name": "Capture The Wool", "command": "/play prototype_pvp_ctw", "enabled": false },
+				{ "name": "King of the Hill 6v6", "command": "/play prototype_koth_two_team", "enabled": false },
+				{ "name": "King of the Hill 3v3v3v3", "command": "/play prototype_koth_four_team", "enabled": false },
```
Comment (SLLCoding) [inline quickplay-games.json:226]: Disabled or removed?

File: quickplay-games.json [hunk 3]
```diff
@@ -1,241 +1,231 @@
...
+				{ "name": "Solo SkyWars", "command": "/play duels_sw_duel", "enabled": true },
+				{ "name": "Doubles SkyWars", "command": "/play duels_sw_doubles", "enabled": true },
+				{ "name": "Solo Bow", "command": "/play duels_bow_duel", "enabled": true },
+				{ "name": "Solo UHC", "command": "/play duels_uhc_duel", "enabled": true },
+				{ "name": "Doubles UHC", "command": "/play duels_uhc_doubles", "enabled": true },
+				{ "name": "Teams UHC", "command": "/play duels_uhc_four", "enabled": true },
+				{ "name": "Deathmatch UHC", "command": "/play duels_uhc_meetup", "enabled": true },
+				{ "name": "Solo NoDebuffs", "command": "/play duels_potion_duel", "enabled": true },
+				{ "name": "Solo Combo", "command": "/play duels_combo_duel", "enabled": true },
+				{ "name": "Solo Potion", "command": "/play duels_potion_duel", "enabled": false },
```
Comment (SLLCoding) [inline quickplay-games.json:171]: Disabled or removed?

File: quickplay-games.json [hunk 4]
```diff
@@ -1,241 +1,231 @@
...
 		{
 			"name": "Prototype",
 			"icon": "https://hypixel.net/styles/hypixel-uix/hypixel/game-icons/Prototype-64.png",
 			"modes": [
-				{ "name": "Capture The Wool", "command": "/play prototype_pvp_ctw" },
-				{ "name": "King of the Hill 6v6", "command": "/play prototype_koth_two_team" },
-				{ "name": "King of the Hill 3v3v3v3", "command": "/play prototype_koth_four_team" },
-				{ "name": "TowerWars Solo", "command": "/play prototype_towerwars_solo" },
-				{ "name": "TowerWars Teams", "command": "/play prototype_towerwars_team_of_two" }
+				{ "name": "Capture The Wool", "command": "/play prototype_pvp_ctw", "enabled": false },
```
Comment (SLLCoding) [inline quickplay-games.json:224]: Need to find new /play command (Capture the Wool moved to Arcade)

**Comment by Ic0vid** [comment]
this is really cool, I don't think the latest quickly can even update like this

**Pull request merged**